src/systemctl/systemctl-logind.c

```diff
@@ -308,24 +307,16 @@ int logind_schedule_shutdown(void) {
         switch (arg_action) {
         case ACTION_HALT:
                 action = "halt";
-                log_action = "Shutdown";
                 break;
         case ACTION_POWEROFF:
                 action = "poweroff";
-                log_action = "Shutdown";
                 break;
         case ACTION_KEXEC:
                 action = "kexec";
-                log_action = "Reboot via kexec";
-                break;
-        case ACTION_EXIT:
-                action = "exit";
-                log_action = "Shutdown";
```

Comment:
Maybe, it is better to drop `ACTION_EXIT` in a separate commit.

Reply:
Right. I'll put it back in for now instead. There's more cleanup to do in that area